Documentation/design/architecture.md

```diff
@@ -140,3 +135,16 @@ Package {name}
   |
   +-- Channel {name} --> CSV {version}
 ```
+
+## Operator Group design
+An operator group consists of a group of namespaces as specified by the label selector in the spec. There are two subsets within this selection that needs distinguishing: the operator namespace where the operator actually runs in and the target namespace(s) which are namespaces the operator have permissions to operate in. Note that the operator namespace is considered part of the operator group's target namespaces even if that particular namespace does not match the label selector.
```

Comment:
> An operator group consists of a group of namespaces as specified by the label selector in the spec. There are two subsets within this selection that needs distinguishing: the operator namespace where the operator actually runs in and the target namespace(s)

To me this reads like the operator namespace is selected by the label selector. I thought the operator namespace was implicitly the `OperatorGroup`'s `ObjectMeta.Namespace`.

Reply:
Ok see what you think now.

Reply:
awesome!